creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -135,6 +161,7 @@ const updateReplicaSetJobProcessor = async function ({
         logger
       ))
   } catch (e: any) {
+    result = 'failure_to_update_replica_set'
```

Comment:
would be good to break the above `try` block into 2 so we can have 2 diff results `failure_determine_new_replica_set` and `failure_issue_update_replica_set`